docs/en/operations/settings/settings_users.md

```diff
@@ -54,27 +54,27 @@ Password could be specified in plaintext or in SHA256 (hex format).
 
 ### user_name/networks
 
-List of networks which the user can connect to ClickHouse server from.
+List of networks that the user can connect to the ClickHouse server from.
 
-Each element of list has one of the following forms:
+Each element of the list has one of the following forms:
 
-- `<ip>` — IP-address or a network mask.
+- `<ip>` — IP address or network mask.
 
     Examples: `213.180.204.3`, `10.0.0.1/8`, `10.0.0.1/255.255.255.0`, `2a02:6b8::3`, `2a02:6b8::3/64`, `2a02:6b8::3/ffff:ffff:ffff:ffff::`.
 
 - `<host>` — Hostname.
 
     Example: `server01.yandex.ru`.
 
-    To check access, DNS query is performed, and all returned IP-addresses are compared to peer address.
+    To check access, a DNS query is performed, and all returned IP addresses are compared to the peer address.
 
 - `<host_regexp>` — Regular expression for hostnames.
 
     Example, `^server\d\d-\d\d-\d\.yandex\.ru$`
 
-    To check access, [DNS PTR query](https://en.wikipedia.org/wiki/Reverse_DNS_lookup) is performed for peer address and then regexp is applied. Then, for result of PTR query, another DNS query is performed and all received addresses compared to peer address. Strongly recommended that regexp is ends with $
+    To check access, a [DNS PTR query](https://en.wikipedia.org/wiki/Reverse_DNS_lookup) is performed for the peer address and then regexp is applied. Then, for the results of PTR query, another DNS query is performed and all received addresses are compared to the peer address. We strongly recommend that regexp ends with $.
```

Comment:
Please correct to `and then the specified regexp is applied` in the first sentence.

For the second sentence can phrased as `Then, another DNS query is performed for the results of the PTR query and all the received addresses are compared with the peer address.`